TailLight/vfeature.h

```diff
@@ -26,13 +26,13 @@ class WDFIOTARGET_Wrap {
 };
 
 
-NTSTATUS SetFeatureColor (
+NTSTATUS SetFeatureColor(
```

Comment:
Why are you introducing whitespace changes to this file?

These changes increases the size of the PR in a unnecessary way. This both increases the review burden and adds risk of regressions.

Reply:
Please don't resolve this comment before it have been properly addressed.